src/unittest/minimizer_mapper.cpp

```diff
@@ -1020,12 +1020,14 @@ TEST_CASE("MinimizerMapper can make correct anchors from minimizers and their zi
                 // Set up to get all the transitions between anchors in the zip code tree
                 auto transition_iterator = algorithms::zip_tree_transition_iterator(seeds, zip_forest.trees.at(0), std::numeric_limits<size_t>::max(), std::numeric_limits<size_t>::max());
                 // And get them
-                transition_iterator(anchors, distance_index, graph, std::numeric_limits<size_t>::max(), [&](size_t from_anchor, size_t to_anchor, size_t read_distance, size_t graph_distance) {
+                transition_iterator(anchors, distance_index, graph, std::numeric_limits<size_t>::max(), [&](vg::algorithms::transition_info transition) {
```

Comment:
If you make the transition be passed by reference, this would need to change type.